tests-remote/sshd-server.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# temporary directory where the sshd pid file will be kept. destroyed on entry and exit
```

Comment:
temporary directory where the dummy server files are kept, like certificates and sshd pid file
destroyed on entry and exit